JenkinsfileRT

```diff
@@ -24,7 +24,7 @@ bc.conda_channels = ['http://ssb.stsci.edu/astroconda']
 bc.conda_packages = ['python=3.6',
                      'cfitsio',
                      'pkg-config',
-                     'pytest',
+                     'pytest=3.8.2',
```

Comment:
Can we be more flexible and do `pytest<=3.8.2` here? I usually avoid `=` unless absolutely necessary.

Reply:
> Hmm. Related or not, what does this mean?
> 
> ```
> equests.exceptions.SSLError: HTTPSConnectionPool(host='bytesalad.stsci.edu', port=443):
> Max retries exceeded with url: /artifactory/scsb-hstcal/dev/wfc3/ir/input/iabg21020_asn.fits
> (Caused by SSLError(SSLError("bad handshake: SysCallError(-1, 'Unexpected EOF')",),))
> ```
Unlikley to be related, but I don't know what's going on here other than an obvious problem with communicating to Artifactory.

We know `pytest 3.8.2 works. I cannot necessarily say the same for other versions.
Unless pytest 3.8.2 suddenly goes missing from the conda main channel, the `<` won't have an effect. Ultimately, this is temporary, so it's not critical either way. Adjusted.